meteor/client/ui/SegmentTimeline/Renderers/DefaultLayerItemRenderer.tsx

```diff
@@ -52,7 +53,16 @@ export class DefaultLayerItemRenderer extends CustomLayerItemRenderer<IProps, IS
 							ref={this.setLeftLabelRef}
 							style={this.getItemLabelOffsetLeft()}
 						>
-							<span className="segment-timeline__piece__label">{this.props.piece.instance.piece.name}</span>
+							<span
+								className={ClassNames(
+									'segment-timeline__piece__label',
+									'with-duration',
```

Comment:
Why does this add the `with-duration` class, even though `this.renderDuration()` can choose not to render the duration at all?